backend/penndata/management/commands/get_rodin_events.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from penndata.models import Event
+
+
+RODIN_EVENTS_WEBSITE = "https://rodin.house.upenn.edu"
+HEADERS = {
+    "User-Agent": "Mozilla/5.0 AppleWebKit/537.36 Chrome/91.0.4472.124 Safari/537.36"
```

Comment:
Woah any reason for this? haha

Reply:
some sites block scraping unless you use headers. they're needed for the venture lab one i think, but i put them in for the college houses just in case. should i only keep in the venture lab one?

Reply:
sounds good, thanks! and yes, I think it would probably be best to just put it on the files that require it